docs/phylum_group.md

```diff
@@ -0,0 +1,28 @@
+---
+title: phylum group
+category: 6255e67693d5200013b1fa3e
+hidden: false
+---
+
+Interact with user groups
+
+```sh
+phylum group [OPTIONS] [SUBCOMMAND]
+```
+
+### Options
+`-j`, `--json`
+&emsp; Produce output in json format (default: false)
+
+### Commands
+* [phylum group create](https://docs.phylum.io/docs/phylum_group_create)
+* [phylum group list](https://docs.phylum.io/docs/phylum_group_list)
```

Comment:
Do we have a convention for indicating that this is the default subcommand?

Reply:
We don't currently, I have tried to just capture that via an example for those top level commands.